### ISSUE TITLE
Unipipe cli on MacOS cannot be started

### ISSUE THREAD

**Issue by tszebeni** [meshcloud/unipipe-service-broker#56]
MacOS has a security feature to require binaries to be signed by known developer accounts. Please see screensot

**Comment by tszebeni** [comment]
<img width="1088" alt="Screenshot 2021-11-29 at 12 44 41" src="https://user-images.githubusercontent.com/328690/143862704-b5e5f08d-2aad-4c08-84ae-aa0c042d5bb2.png">

**Comment by tszebeni** [comment]
![Uploading Screenshot 2021-11-29 at 12.48.53.png…]()

**Comment by JohannesRudolph** [comment]
@tszebeni thanks for reporting in - we investigated [code signing for collie-cli](https://github.com/meshcloud/collie-cli/issues/27) but at the moment are blocked by an upstream issue in deno https://github.com/denoland/deno/issues/11154

**Comment by DorukAkinci** [comment]
@tszebeni, As a small workaround for now, you can right click the application and press `Open`. Macos will ask that Will you trust this application and click Yes. Afterwards there will be no issue anymore.

**Comment by tszebeni** [comment]
Thanks @JohannesRudolph and @DorukAkinci .

**Comment by DorukAkinci** [comment]
It seems we can also sign the applications with a terminal command. Therefore this manual step can be much more smoother if we can implement something like that. 

The following command signs the application with an ad-hoc signing identity, so it should have the required trust. `codesign --force --deep -s - $(which unipipe)` however i could not fully test it because i've already done the method that i shared above.
